### DISCUSSION TITLE
Video restarts when touching forward or reverse buttons

### DISCUSSION THREAD

**Discussion by HorlyVera** [muxinc/media-chrome#847] [Q&A]
Hi, I have been working with the library and at first the forward or back buttons worked fine, however in the last weeks when using these buttons instead of forward or rewind, the video restarts both on Web and Android (On iOS it works fine). I did a test with one of the examples that come in the official documentation and it works correctly, but when I make the change of url to my video no longer works, it is strange because I do not think it is an encoding problem since I am also using an mp4 format, do you know what could it be?

**Comment by luwes** [comment]
my hunch is still video encoding, if there are not enough keyframes for example. mp4 is just tip of the iceberg, there can be many different things in that video container. do you have an example online?

**Reply by HorlyVera** [reply]
First of all thank you very much for your answer. I share with you one of the video url's of the website
https://api.mind2.me/content/video/84_Felicidad_1.mp4
Do you need an example of the implementation? or is the video link enough?
